.github/workflows/aws-docker-deploy.md

```diff
@@ -18,7 +18,9 @@ To use this GitHub Action, you need to define it in your workflow file (e.g., `.
 
 4. `dotenv-value` (optional, type: string): The value needed to fill the `.env` file for the front-end client. This input is only relevant if `client-environment-file-path` is provided.
 
-5. `npm-read-packages-token` (optional, type: string): Token for reading NPM packages from private repositories
+### Secrets
+
+1. `NPM_READ_PACKAGES_TOKEN` (optional, type: string): Token for reading NPM packages from private repositories
```

Comment:
Probably not optional right ?

Reply:
I guess it is if you're not using any private packages, nvm

Reply:
yes it is, some builds do not require it and it still work. Have to handle this since it's a generic action